docs/Install-with-Docker.md

```diff
@@ -72,19 +72,17 @@ You need to clone the AzerothCore repository (or use your own fork):
 git clone https://github.com/azerothcore/azerothcore-wotlk.git
 ```
 
-Now cd into the main directory using `cd azerothcore-wotlk`. **All commands will have to be run from this position**.
+Now go into the main directory using `cd azerothcore-wotlk`. **All commands will have to be run inside this folder**.
 
 ### WoW Client Data files
 
 You also need to have the data files. Check the step "5) Download the data files" from the [installation guide](Installation#5-download-the-data-files).
 
-Put your data files into the `docker/worldserver/data/` folder that is inside `azerothcore-wotlk`.
+Put your data files into `docker/worldserver/data/` which is inside `azerothcore-wotlk`.
 
 ### Installation
 
-Inside your terminal (if you use Windows, use git bash), run the following commands.
-
-**Note for Windows users**: when running the following commands, you may need to use `bash bin\acore-...` instead of `./bin/acore-...`. Note using `\` instead of `/` and `bash bin...` instead of `./bin...`
+Inside your terminal (if you use Windows, use git bash), run the following commands inside azerothcore-wotlk'
```

Comment:
why deleting the note? I've specifically added this after a feedback from a Windows user... the rest looks good!

Reply:
"Inside your terminal (if you use Windows, use git bash), run the following commands."
this line implies you run the bash stuff again inside the terminal where you've already ran bash. bit weird imo since the rest of the guide uses bash when this line expects you use to use command prompt

Reply:
if you are on Windows and think this is enough, fine for me. I don't use Windows so I can't test myself